.changeset/tasty-elephants-develop.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+Internal refactor to remove `es-module-lexer` and support `wrangler types` for Workers with Durable Objects & JSX
```

Comment:
what was wrong with wrangler types + durable objects specifically?

Reply:
`es-module-lexer` was failing to parse JSX, but was only run when Durable Object bindings were used

Reply:
ah right I read that wrong, makes sense 👍